scipy/special/add_newdocs.py

```diff
@@ -3376,19 +3376,23 @@ def add_newdoc(name, doc):
 
 add_newdoc("hyp0f1",
     r"""
-    hyp0f1(v, x)
+    hyp0f1(v, x, out=None)
```

Comment:
It looks like the parameter named `x` in this line is designated as `z` in the docstring, including the original function / summation definition.

Reply:
Good catch; fixed it.